canonical-web-and-design/maas-ui
# refactor(legacy): remove unused legacy auth

## Done

- Remove the code for handling external auth from the legacy client, this has been unused for some time. This reduces the built bundle by about 25%.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Login using external auth (see me if you don't know how to do this) and visit the legacy client, everything should work as it did.
